app/components/Views/confirmations/hooks/gas/useFeeCalculations.test.ts

```diff
@@ -23,6 +29,17 @@ jest.mock('../../utils/token', () => ({
 }));
 
 describe('useFeeCalculations', () => {
+  const mockIsTestNet = jest.mocked(isTestNet);
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    mockIsTestNet.mockReturnValue(false);
+  });
+
+  afterEach(() => {
+    mockIsTestNet.mockReturnValue(false);
+  });
+
```

Comment:
```suggestion
  beforeEach(() => {
    jest.resetAllMocks();
  });

```

I believe we can remove the mockReturnValues here
1) no need to add to both beforeEach and afterEach
2) jest.resetAllMocks() clears the mock calls (.mock.calls, .mock.results) and resets to the mock implementation defined in jest.mock() above

Reply:
Good call - just a small leftover while trying to make it locally - fixed now.